spec/cirro_io_v2/resources/user_spec.rb

```diff
@@ -0,0 +1,21 @@
+RSpec.describe CirroIOV2::Resources::User do
+  describe '#find' do
+    subject(:find_user) { client.User.find(id) }
+
+    let(:user) { FactoryBot.create(:user) }
+    let(:id) { user.id }
+    let(:client) { CirroIOV2::Client.new }
+
+    before { allow(client.request_client).to receive(:request).and_return(OpenStruct.new(body: user.to_h)) }
```

Comment:
This is a terrible way to test a client that does HTTP request. 
The test should have no understanding of the implementation

We should use webwock, if you need with this part I can help re-writing some of the test